core/templates/pages/exploration-player-page/services/hints-and-solution-manager.service.spec.ts

```diff
@@ -112,9 +82,9 @@ describe('HintsAndSolutionManager service', function() {
     expect(hasms.isHintConsumed(1)).toBe(false);
 
     // For releaseHint.
-    $timeout.flush();
+    tick(60000);
```

Comment:
Can we use a constant for the number passed here?

Reply:
Ditto below

Reply:
Done